modal/sandbox.py

```diff
@@ -26,17 +27,33 @@
 
 
 class _LogsReader:
-    """Provides an interface to buffer and fetch logs from a sandbox stream (`stdout` or `stderr`)."""
+    """Provides an interface to buffer and fetch logs from a sandbox stream (`stdout` or `stderr`).
+
+    As an asynchronous iterable, the object supports the async for statement.
+    **Usage**
+
+    ```python notest
+    sandbox = stub.app.spawn_sandbox("bash", "-c", "while true; do echo foo; sleep 1; done")
+    async for message in sandbox.stdout:
+        print(f"Message: {message}")
+    ```
+    """
 
     def __init__(self, file_descriptor: int, sandbox_id: str, client: _Client) -> None:
         """mdmd:hidden"""
 
         self._file_descriptor = file_descriptor
         self._sandbox_id = sandbox_id
         self._client = client
+        self._stream = None
+        self._last_log_batch_entry_id = ""
+        # Whether the reader received an EOF. Once eof is True, it returns
```

Comment:
```suggestion
        # Whether the reader received an EOF. Once EOF is True, it returns
```

nit: consistency within module (and indeed within same comment block)